lib/eventception/dispatcher.rb

```diff
@@ -98,20 +98,20 @@ def listeners_for?(event_name:)
     # priority::
     #   The higher this value, the earlier an event listener will be triggered in the chain (defaults to 0)
     #
-    def add_listener(event_name:, listener:, priority: 0)
-      event_listeners[event_name][priority] << listener
+    def add_listener(event_name:, listener_handler:, priority: 0)
+      event_listeners[event_name][priority] << listener_handler
       sorted.delete(event_name)
 
-      listener
+      listener_handler
```

Comment:
Hmm not sure if it would still make sense to return the listener if we ended up not exposing the handler via the interface. Maybe `nil` in that case.